libmuscle/cpp/src/libmuscle/receive_timeout_handler.cpp

```diff
@@ -9,16 +11,22 @@ ReceiveTimeoutHandler::ReceiveTimeoutHandler(
     , peer_instance_(peer_instance)
     , port_name_(port_name)
     , slot_(slot)
-    , timeout_(timeout) {}
+    , timeout_(timeout)
+    , num_timeout_(0) {}
 
 double ReceiveTimeoutHandler::get_timeout()
 {
-    return timeout_;
+    // Increase timeout by a factor 1.5 with every timeout we hit:
+    return timeout_ * std::pow(1.5, (double)num_timeout_);
```

Comment:
Should `num_timeout_` be incremented here? Or somewhere else?

Reply:
It's also in the C++ one, but only in added in a later commit when I found that the tests failed ;) https://github.com/multiscale/muscle3/pull/299/commits/f3f8e317acdb169a57ba398369bc6ef9ad52ec50#diff-7c147a535e0c6698203173d19156e7070c502df13dc79087da799e7bdc2113b6

Reply:
Ah, I reviewed one commit at a time. Good test of the tests and also of the review :smile: :.